engine/class_modules/monk/sc_monk.cpp

```diff
@@ -1767,6 +1771,16 @@ struct blackout_kick_t : overwhelming_force_t<charred_passions_t<monk_melee_atta
       p()->gain.bok_proc->add( RESOURCE_CHI, base_costs[ RESOURCE_CHI ] );
   }
 
+  int n_targets() const override
+  {
+    // The Ancient Concordance buff's spell data modified number of targets hit, but the value is 1 instead of 3, so it
+    // is useless. We set the value manually to 3.
+    if ( p()->buff.ancient_concordance->check() )
+      return 3;
```

Comment:
It's not necessarily useless, as it may provide 2 stacks in game. Note that the effect has the "Aura Points on Client" bit set, which has to do with behaviours related to effect stacking.

Would need to perform some in game science, as it is also not logged.